tamr_toolbox/utils/upstream.py

```diff
@@ -96,6 +97,79 @@ def _find_projects(
         return upstream_projects
 
 
+def _find_datasets(dataset: Dataset) -> List[Dataset]:
+    """Returns a dataset's upstream datasets.
+
+        Args:
+            dataset: a tamr dataset
```

Comment:
is this a tamr Dataset object? I think it could be defined more clearly. 
```suggestion
            dataset: a tamr Dataset Object
```